lib/utils/getLocales/index.ts

```diff
@@ -0,0 +1,48 @@
+import { en, fr, de, ru, zh, ko, es, hi } from "make-plural/plurals";
+import { IS_PRODUCTION } from "../../constants";
+
+// TODO: remove NODE_ENV=test hack from package.json https://github.com/lingui/js-lingui/issues/433
+// Define locales
+interface ILocales {
+  [locale: string]: {
+    plurals: (
+      n: number | string,
+      ord?: boolean
+    ) => "zero" | "one" | "two" | "few" | "many" | "other";
+    time: string;
+    label: string;
+    production_ready: boolean;
+  };
+}
+
+const locales: ILocales = {
```

Comment:
how about `productionLocales` and `stagingLocales` defined as separate objects? just makes it more clear at a glance